docker-compose.yaml

```diff
@@ -54,3 +54,4 @@ volumes:
 
 networks:
   mqsnet:
+    name: mqsnet
```

Comment:
By default, it creates some unpredictable prefix, so I decided to define it explicitly to make compatible with upload script